spec/features/test_page_spec.rb

```diff
@@ -1,6 +1,34 @@
+
 feature 'test page' do
   scenario 'visit test page' do
     visit '/test'
     expect(page).to have_content('test page')
   end
+
+  scenario 'player can add their name for a personalised game' do
+    visit "/"
+    fill_in('player_1', with: 'Player 1')
+    click_on('Play Game')
+    expect(page).to have_content "Player 1 vs. CPU"
+  end
+
+  scenario 'player can click either rock, paper, or scissors buttons' do
+    visit "/"
+    fill_in('player_1', with: 'Player 1')
+    click_on('Play Game')
```

Comment:
could you webhelper for this code to save on repeating for each test.

Reply:
Will have to try to learn how to use these :)